cluster/cluster.go

```diff
@@ -50,14 +50,16 @@ type NodeEntry struct {
 
 // ClusterInfo is the basic info about the cluster and its nodes
 type ClusterInfo struct {
-	Size          int
-	Status        api.Status
-	Id            string
-	NodeEntries   map[string]NodeEntry
-	LoggingURL    string
+	Size        int
+	Status      api.Status
+	Id          string
+	NodeEntries map[string]NodeEntry
+	// LoggingURL - Deprecated
+	LoggingURL string
```

Comment:
I think we can get rid of these too. unMarshal should skip fields that don't exist in the object i believe.
If not it's fine to leave these.